src/python/pants/option/global_options.py

```diff
@@ -46,10 +47,21 @@ class GlobMatchErrorBehavior(Enum):
 class FileNotFoundBehavior(Enum):
     """What to do when globs do not match in BUILD files."""
 
+    ignore = "ignore"
     warn = "warn"
     error = "error"
 
     def to_glob_match_error_behavior(self) -> GlobMatchErrorBehavior:
+        deprecated_conditional(
+            lambda: self == self.ignore,
+            removal_version='1.28.0.dev2',
+            entity_description='--files-not-found-behavior-option=ignore',
+            hint_message="If you currently set `--files-not-found-behavior=ignore`, you will "
+            "need to instead either set `--files-not-found-behavior=warn` (the "
+            "default) or `--files-not-found-behavior=error`. Ignoring when files are "
+            "not found often results in subtle bugs, so we are removing the option.",
+            deprecation_start_version='1.27.0.dev2',
```

Comment:
This can be 1.27.0.dev0 if you'd like. We clarified the policy to explain that you can always start deprecations in dev0, even if you wrote it when on something like dev2.